vercel-functions/prerender/_prerender-ember-route.func/parse-prerender-path.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Converts a Request URL used to access a Prerender Function into a proper URL
+ * for pre-rendering an Ember route with FastBoot.
+ *
+ * Strips `/prerender` from the start of the path, since it's the entry point
+ * to Prerender Functions.
+ *
+ * Functions whose name are wrapped in [] are considered "wildcard" functions,
+ * in which case a corresponding dynamic segment with same name should be present
+ * in the Vercel rewrite rule. The wildcard function name in the path will be
+ * replaced with the value of the dynamic segment. Wildcard function names can
+ * contain multiple dynamic segments, such as: [user][subpage]
+ *
+ * @param {string} url Vercel Request URL received by the Prerender Function.
+ * @returns {string} Resolved Ember path to pre-render using FastBoot.
+ * @example
+  Considering the following Vercel rewrite rules:
+  ```
+  /demo/code-mirror -> /prerender/demo/code-mirror
+  /users/:user -> /prerender/users/[user]
+  /concepts/:concept -> /prerender/concepts/[concept]
+  /examples/:page/:subpage -> /prerender/examples/[page][subpage]
+  /test -> /prerender/test
+  ```
+  Making requests below will result in corresponding Request URLs received by
+  the Prerender Function, and `parsePrerenderPath` will produce the following
+  FastBoot paths for such Request URLs:
+  ```
+  /demo/code-mirror -> /prerender/demo/code-mirror -> /demo/code-mirror
+  /users/abcde -> /prerender/users/[user]?user=abcde -> /users/abcde
+  /concepts/xyzqwe -> /prerender/concepts/[concept]?concept=xyzqwe -> /concepts/xyzqwe
+  /examples/abc/def -> /prerender/examples/[page][subpage]?page=abc&subpage=def -> /examples/abc/def
```

Comment:
```suggestion
  /examples/abc/def -> /prerender/examples/[page]/[subpage]?page=abc&subpage=def -> /examples/abc/def
```

(Let me know if this is incorrect)

Reply:
Oh, okay I see what's happening here - the last part **needs** to not have the `/` for this to work, `parsePrerenderPath` assumes this.

Reply:
Exactly! As far as I understood a vercel `something.func` can not contain any more nested `other.func` functions